README.md

```diff
@@ -1,26 +1,97 @@
+[![Download](https://api.bintray.com/packages/ssed-oss-jcenter/ssed-mobile-libs/android-miniapp/images/download.svg)](https://bintray.com/ssed-oss-jcenter/ssed-mobile-libs/android-miniapp/_latestVersion)
 [![CircleCI](https://circleci.com/gh/rakutentech/android-miniapp.svg?style=svg)](https://circleci.com/gh/rakutentech/android-miniapp)
 [![codecov](https://codecov.io/gh/rakutentech/android-miniapp/branch/master/graph/badge.svg)](https://codecov.io/gh/rakutentech/android-miniapp)
 [![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](https://opensource.org/licenses/MIT)
 
 # MiniApp SDK for Android
 
 Provides a set of tools and capabilities to show mini app in Android Applications. The SDK offers features like fetching, caching and displaying of mini app. 
-For instructions on implementing in an android application, see the [User Guide](miniapp/USERGUIDE.md).
+For instructions on implementing in an android application, see the [User Guide](https://rakutentech.github.io/android-miniapp/).
 
 ## How to build
 
 This repository uses submodules for some configuration, so they must be initialized first.
 
 ```bash
-$ git submodule init
-$ git submodule update
-$ ./gradlew assemble
+$ git submodule update --init
 ```
 
-## How to test the Sample app
+Next, you must define a few settings used by the Sample App. These can be defined either as environment variables or Gradle properties (in your `~/.gradle/gradle.properties` file).
 
-We are still working on this, please watch this space for future updates.
+```
+MINIAPP_SERVER_BASE_URL=https://www.example.com/
+HOST_APP_ID=test-host-app-id
+HOST_APP_SUBSCRIPTION_KEY=test-subs-key
+HOST_APP_UA_INFO=MiniAppDemoApp/1.0.0
```

Comment:
Because we add `HOST_APP_UA_INFO` as an example here so we should include it in the example of production config too. I also noticed that we use the same `HOST_APP_UA_INFO` in build.gradle for `stg` and `prod`. We can update it to `HOST_APP_PROD_UA_INFO` in `prod`.

Reply:
Yeah, that's a good point. I have added `HOST_APP_PROD_UA_INFO` as a config option.